sprocs/assessment_questions_calculate_stats.sql

```diff
@@ -10,17 +10,15 @@ relevant_assessment_instances AS (
         assessment_questions AS aq
         JOIN assessments AS a ON (a.id = aq.assessment_id)
         JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
-        LEFT JOIN groups AS g ON (g.id = ai.group_id AND g.deleted_at IS NULL)
-        LEFT JOIN group_users AS gu ON (gu.group_id = g.id)
-        JOIN enrollments AS e ON ((e.user_id = ai.user_id OR e.user_id = gu.user_id) AND e.course_instance_id = a.course_instance_id)
```

Comment:
These were only being used to call `users_is_instructor_in_course_instance()` so we can simply drop them now that we are storing `include_in_statistics` in the assessment instance.